.github/workflows/build.yml

```diff
@@ -137,6 +137,8 @@ jobs:
         context: .
         tags: ghcr.io/ls1intum/artemis:${{ steps.compute-tag.outputs.result }}
         push: true
+        cache-from: type=gha
+        cache-to: type=gha,mode=min
```

Comment:
Did you also try `max` cache mode? Could help speed up w/ intermediate layers, e.g. build deps. However, slows down import/export.

Reply:
Hey yes I did and the `min` cache mode worked better in our case at least when I tested it.